examples/.ipynb_checkpoints/example_1 - Loading_a_Bill_of_Materials-checkpoint.ipynb

```diff
@@ -20,6 +20,7 @@
    "source": [
     "import json\n",
     "from pathlib import Path\n",
+    "\n",
```

Comment:
Should checkpoint files like this be kept? I thought you deleted a whole bunch of them.

Reply:
The addition to the .gitignore and deletion are in the develop branch. When that is merged in it will be captured but i could remove as part of this commit?

Reply:
No, that's fine. Wasn't looking closely enough at the respective branches the changes were made to.